### PR TITLE
Adding a filter to su_theme_update_row()

### PR DESCRIPTION
**Pull request by afragen**
This will allow for insertion of some data if the theme doesn't live in .org

### CODE REVIEW

File: src/update.php [hunk 1]
```diff
@@ -182,6 +182,16 @@ function su_theme_update_row( $theme_key, $theme ) {
 		'height'    => 800,
 	), $current->response[ $theme_key ]['url'] );
 
+	/**
+	 * Filter $details_url for themes not in .org repo.
```
Comment (swissspidy) [inline src/update.php]: I would change the wording here, e.g. "Filters the details URL for theme update information". Not the third-person singular verb.

**Comment by swissspidy** [comment]
Hey there

Thanks for your PR. `wp_theme_update_row()` currently doesn't have such a filter, so it might be better to discuss it in a trac ticket.

Also, I'm not sure if this filter alone makes sense in the first place. When your theme or plugin isn't hosted on WordPress.org, you need to filter much more than just the details URL. You basically need to roll your own API checks for updates. Therefore you'll already have custom data in the transient fetched via `get_site_transient( 'update_themes' )`.

**Comment by afragen** [comment]
I created a trac ticket, https://core.trac.wordpress.org/ticket/36923

I'll close here.

**Comment by afragen** [comment]
> Also, I'm not sure if this filter alone makes sense in the first place. When your theme or plugin isn't hosted on WordPress.org, you need to filter much more than just the details URL. You basically need to roll your own API checks for updates. Therefore you'll already have custom data in the transient fetched via get_site_transient( 'update_themes' ).

I already do this, but it's not capable of being displayed without a modification here. Admittedly a small use case, but a use case. Also, resizing the window is now possible.

**Pull request closed without merging**